pkgs/by-name/pi/pinit/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  pkg-config,
+  vala,
+  blueprint-compiler,
+  wrapGAppsHook4,
+  libadwaita,
+  libgee,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "pinit";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "ryonakano";
+    repo = "pinit";
+    rev = finalAttrs.version;
+    hash = "sha256-BL5ZzUtjsvl7Vw1P53hZr2LRW2WI8ls/bjdFN5YXtTI=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    vala
+    blueprint-compiler
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [
+    libadwaita
+    libgee
+  ];
+
+  meta = {
+    description = "Pin portable apps to the launcher";
+    homepage = "https://github.com/ryonakano/pinit";
+    license = with lib.licenses; [ gpl3Plus cc0 ];
+    mainProgram = "com.github.ryonakano.pinit";
```

Comment:
Is this really the name of the main program ?

Reply:
Yes, some gtk developers like this rdns style

```
=> tree ./result/bin
./result/bin
└── com.github.ryonakano.pinit
```

Reply:
Shouldn't we rename that to `pinit` ? How is it done in other distributions?

Reply:
CC @jtojnar @bobby285271: What's your opinion on this question?

Reply:
Renaming can have side-effects in some cases (e.g. `WmClass` and finding resources). We could add a symlink but it is probably pointless. People will likely launch it using desktop file anyway.

Reply:
So, do we have a consensus here ?

Reply:
> So, do we have a consensus here ?

I don't think this is necessary - we already provide mainProgram for the case where the executable is different from the package name. We should respect upstream's decision.